src/PowerShellEditorServices/Console/UnixConsoleOperations.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using UnixConsoleEcho;
+
+namespace Microsoft.PowerShell.EditorServices.Console
+{
+    internal class UnixConsoleOperations : IConsoleOperations
+    {
+        private const int LONG_READ_DELAY = 400;
```

Comment:
Does this feel laggy? `300` is probably fine and would help feel less laggy.

Reply:
I started with `500` which did feel laggy.  `400` didn't feel laggy, but that was through VNC so not super accurate.  Either way, I'm definitely more comfortable with `300`, I'll get that in.  Thanks for your help on this! 😄

Reply:
Switched to 300ms. The first delay can still feel a *little* laggy if you type particularly fast and catch it right at the start of the wait, but I don't think it'll be all that noticeable if you aren't looking for it.  At least good enough until the issue is fixed in corefx.